### DISCUSSION TITLE
Table entering issue

### DISCUSSION THREAD

**Discussion by marinnkobzar** [AntlerVC/firetable#1402] [Support / Q&A]
**Describe the bug**

It is not possible to enter the table. I was given all possible permissions, I also tried to log in from my colleague's account, in this case I can't see the table as well. I tried to log in from different browsers, anonymous tabs, to reboot the laptop.
I get this error when I try to accept an invitation to the project: Could not respond to invite: duplicate key value violates unique constraint “invites_email_workspaceId_state_key”. 
Also, we tried to send to me invitations to different emails, and in this case, I still don't see the table that I need on the project.
When I still managed to accept the invitation to the project, my colleagues sent me a link to the table I needed, and I still could not see this table, and saw the following message: Table not found
Make sure you have the right ID "deals". - Despite the fact that all my colleagues see this table.

**Expected behavior**
1. I received an invitation to join the project on Rowy
2. I accept the invitation, go to the project and see the table that all other team members see there

**Screenshots**
<img width="1463" alt="Screenshot 2023-09-04 at 12 33 13" src="https://github.com/rowyio/rowy/assets/144008961/1a2d9412-e1e3-4f81-a589-d9f0c2982160">

<img width="1463" alt="Screenshot 2023-09-04 at 12 42 04" src="https://github.com/rowyio/rowy/assets/144008961/bd3a29f1-bd1f-456b-b788-0168ccbdcf9f">

**Rowy Run version**
If applicable, share the Rowy Run version from your project settings.

**Desktop (please complete the following information):**
 - OS: iOS
 - Browser: chrome and safari
 - Version: latest

**Additional context**

**Comment by Chadha93** [comment]
Hi @marinnkobzar, there could be a few reasons for this: either the member you invited is already in the workspace, or this might be some glitch.

Could you share the following information to help us proceed with the investigation?
1. Your Firebase Project ID
2. Your Email Id
3. The Team Member's Email ID
Also, check if the member already exists in your Workspace.


I invite you to join our [Discord](https://www.rowy.io/discord) and post this on the community support channel. 

--
Cheers,
Gaurav Chadha | Technical Support Engineer | [Rowy](https://rowy.app/) + [BuildShip](https://buildship.com/)
backend for everyone

**Comment by Chadha93** [comment]
Closing this discussion, feel free to re-open or post on Rowy Discord with project details. https://www.rowy.io/discord.
